README.md

```diff
@@ -1,3 +1,47 @@
 # conda-auth
 
-A conda plugin for handling multiple authentication schemes
+A conda plugin for handling various HTTP based authentication schemes.
+
+Conda auth is a plugin that handles various authentication schemes as well as login session management. Currently, the following authentication schemes are supported:
+
+- HTTP Basic Authentication
+
+On top of this, conda-auth supports session management via two subcommands for logging into services (conda auth login) and logging out of services (conda auth logout).
+
+## Installation
+Install `conda-auth` into your base environment from the `conda-forge` channel:
+```
+conda install --name base --channel conda-forge conda-auth
+```
+
+## Why conda-auth?
+
+Currently when using conda, if you want use a channel that requires the use of HTTP Basic Authentication, you need to store your user credentials in your `.condarc` file:
+
+```
+channels:
+  - https://username:password@example.com/channel
+```
+
+The example above has one primary problem: storing credentials in clear text. This is a practice that is frowned upon generally and one we should avoid if possible.
+
+The conda-auth project aims to solve this problem by storing credentials in an encrypted and password protected manner. This is accomplished by introducing a dependency on the [keyring project](https://github.com/jaraco/keyring). This Python library uses the underlying secret store mechanism for many types of desktop operating systems, including Windows, OSX and Linux.
+
+Using this plugin will then ensure that users are storing their user credentials in a safer manner.
```

Comment:
````suggestion
## Usage

**Log in** to a channel protected by HTTP basic authentication:

```
conda auth login https://example.com/my-protected-channel --username $USERNAME
```

**Log out** of a channel to remove credentials from your computer:

```
conda auth logout https://example.com/my-protected-channel
```

````

I don't think this section should be included on the front page. Instead, let's just include a "usage" section to provide examples of how the commands are used.